og.module

```diff
@@ -171,17 +171,14 @@ function og_entity_create_access(AccountInterface $account, array $context, $bun
     return $access_result;
   }
 
-  // Save some legwork if the user has the core permission and strict node
-  // access is not set.
-
   $config = \Drupal::config('og.settings');
-  $entity_access_strict = $config->get('node_access_strict');
-  // @todo: $entity isn't passed.
-  /*
-  if (!$entity_access_strict && $entity->access('create', $account)) {
+  $entity_access_strict = $config->get('entity_access_strict');
+
```

Comment:
We might need to think about some plugins or other way to declare this for different entity types maybe?